src/System.Net.Primitives/tests/FunctionalTests/LoggingTest.cs

```diff
@@ -25,10 +25,8 @@ public void EventSource_ExistsWithCorrectId()
         }
 
         [Fact]
-        [SkipOnTargetFramework(
-            ~TargetFrameworkMonikers.Netcoreapp, 
-            "NetFramework: NetEventSource is only part of .NET Core;" + 
-            "UapAot: RemoteExecutorConsoleApp.exe crashes trying to load netstandard.dll - StrongName issue")]
+        [ActiveIssue("https://github.com/dotnet/corefx/issues/19909", TargetFrameworkMonikers.Uap)]
```

Comment:
@davidsh I think this discussion is worth splitting into a different issue so that more people can contribute if interested.